gulp/tasks/svg.js

```diff
@@ -57,7 +57,7 @@ gulp.task('svg:sprite', () => {
     }
 
     const sprites = getSprites();
-    return sprites.map(sprite => bundle(sprite));
+    return sprites.forEach(sprite => bundle(sprite));
```

Comment:
Should’t we merge the streams here? Or, is it OK to return `sprites`?
But if it works...
@vbulant